docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+“Soortgelijke gegevens opslaan bij soortgelijke gegevens” is hiervoor een goed uitgangspunt. Zo ontstaan diverse “kernregistraties”: coherente en integere registraties met meervoudig gebruikte soortgelijke gegevens die vanuit verschillende procesapplicaties worden bijgehouden.  Bijvoorbeeld één gemeentelijke kernregistratie medewerkers en niet een (deel-)registratie per afdeling/proces.
+
+Voor het Zaakgericht werken heeft dit als gevolg dat veel objecten die nu in een Zakenmagazijn of procesapplicatie worden bijgehouden (behandelend medewerker, betreffend object, aanvrager (klant),..) vervangen moeten worden door een verwijzing naar de desbetreffende kernregistratie.
+
+### 1.2.3. Vermindering van informatiestromen
+
+In de huidige gemeentelijke informatievoorziening vindt de interactie tussen processen plaats door middel van gegevensuitwisselingsberichten tussen desbetreffende applicaties (cq. referentiecomponenten). De ontvangen informatie wordt veelal ook weer opgeslagen door en in de ontvangende procesapplicatie. Een voorbeeld zijn mutaties van het verblijfsadres van een burger (a.g.v. een verhuizing) die door de GBA- (cq. BRP-)beheerapplicatie verstrekt worden aan bijvoorbeeld een vergunningenapplicatie, belastingenapplicatie en sociaal domeinapplicatie.
+In een gegevenslandschap is er van een dergelijke gegevensdistributie geen sprake meer. Hooguit wordt een proces genotificeerd over een gebeurtenis (zoals een verhuizing). De daarbij betrokken gegevens kunnen vervolgens, indien gewenst, door de ontvangende applicatie opgehaald worden uit de desbetreffende registratie (en niet opnieuw vastgelegd worden) om daarmee de voor het proces relevante afhandeling te kunnen doen.
+Het versturen van berichten met inhoud tussen verschillende referentiecomponenten wordt dus zoveel mogelijk beperkt en vervangen door het sturen van notificaties en het opvragen van de bijbehorende gegevens uit een registratie.
+Specifieke aandacht verdienen aanvragen, meldingen e.d. die ‘op een website’ (of vanuit een App) ingediend worden. Het indienen wordt ondersteund door de referentiecomponent voor het aanvragen van producten en diensten (“e-Formulierencomponent”). In GEMMA 2 wordt een dergelijke aanvraag gerouteerd (in een bericht) naar de van toepassing zijnde generieke of specifieke ZAC. In een gegevenslandschap kan hiervan geen sprake zijn. Voor de hand liggend is dat de e-Formulierencomponent die aanvraag opslaat (vanuit de bron, dus; in een desbetreffende registratiecomponent; kernregistratie verzoeken?), een notificatie stuurt naar de desbetreffende ZAC die naar aanleiding daarvan de behandeling start (en daartoe onder meer de zojuist genoemde registratiecomponent bevraagt).
```

Comment:
Waarom kan een e-formulierencomponent niet direct een zaak aanmaken in een ZRC?
Als er een kernregistratie verzoeken nodig is, waar zit dan de kennis naar welk ZAC er een notificatie gestuurd moet worden?

Reply:
> Niet geremd door enige kennis van de huidige GEMMA opzet maar in mijn beleving stuurt een e-Formulierencomponent geen notificatie naar een ZAC.

volgens GEMMA 2 nog wel, zie: https://www.gemmaonline.nl/index.php/ZGW_in_GEMMA_2_compleet#Indienen_productaanvraag_via_webformulier

Reply:
het antwoord zit volgens mij in verzoeken. zie ook # 390
ingevulde webformulier + attributen opslaan als een melding/verzoek of aanvraag. Nog een manier bedenken om generieke delen van een aanvraag (aanvrager, datum, bijbehorende documenten, etc) en domeinspecifieke zaken goed te kunnen linken.
Dan vanuit ZAC kijken of er nog nieuwe aanvragen klaar staan/evt genotificeerd worden..

verdere discussie aub in #390 of op google docs :-)

Reply:
> Waarom  kan een e-formulierencomponent niet direct een zaak aanmaken in een ZRC?
zie comments hierboven

> Als er een kernregistratie verzoeken nodig is, waar zit dan de kennis naar welk ZAC er een notificatie gestuurd moet worden?
goede vraag. misschien ergens in de buurt van een ZTC?